src/client/java/teammates/client/scripts/scalabilitytests/InstructorFeedbackResultsPageScaleTestDataGenerator.java

```diff
@@ -0,0 +1,171 @@
+        private String startTime = "2012-04-01 11:59 PM UTC";
+        private double timeZone = 8.0;
+    }
+
+    class Student {
+        private String googleId = "CFResultsUiT.";
+        private String comments = "This is a student.";
+        private String course = "CFResultsUiT.CS2104";
+        private String email = "CFResultsUiT.";
+        private String name;
+        private String section = "Section A";
+        private String team = "Team 1";
+
+        Student(String name) {
+            this.name = name;
+            this.googleId += name;
+            this.email += name + ".tmt";
+        }
+
+        public String getGoogleId() {
+            return googleId;
+        }
+    }
+
+    class Response {
+        private String giver;
+        private String recipient;
+        private String feedbackQuestionId;
+        private String courseId = "CFResultsUiT.CS2104";
+        private String feedbackQuestionType = "TEXT";
+        private String feedbackSessionName = "Open Session";
+        private String giverSection = "Section A";
+        private String recipientSection = "Section A";
+        private Metadata responseMetaData = new Metadata("Response.");
+
+        Response(String giver, String recipient, String feedbackQuestionId) {
+            this.giver = giver;
+            this.recipient = recipient;
+            this.feedbackQuestionId = feedbackQuestionId;
+        }
+    }
+
+    private InstructorFeedbackResultsPageScaleTestDataGenerator(int numQuestions, int numStudents) {
+        courses.put("CFResultsUiT.CS2104", new Course());
+        accounts.put("CFResultsUiT.instr", new Account());
+        feedbackSessions.put("Open Session", new Session());
+
+        RandomNameGenerator nameGenerator = new RandomNameGenerator();
+        for (int i = 0; i < numStudents; i++) {
+            String name = nameGenerator.next();
+            students.put(name.replace("_", " "), new Student(name.replace("_", ".")));
+        }
+
+        int count = 0;
+        for (int i = 0; i < numQuestions; i++) {
+            feedbackQuestions.put("question" + i, new Question(i));
+            for (Student giver : students.values()) {
+                for (Student recipient : students.values()) {
+                    feedbackResponses.put(
+                            "response" + count,
+                            new Response(giver.getGoogleId(), recipient.getGoogleId(), String.valueOf(i)));
+                    count++;
+                }
+            }
+        }
+    }
+
+    public static void main(String[] args) throws IOException {
+        int[] studentNums = {10, 20};
+        int[] questionNums = {1, 5, 10};
+        Gson gson = new GsonBuilder().setPrettyPrinting().create();
+        for (int studentNum : studentNums) {
+            for (int questionNum : questionNums) {
+                Writer writer = new FileWriter(
+                        "src/client/java/teammates/client/scripts/scalabilitytests/"
```

Comment:
This is not where `InstructorFeedbackResultsPageScaleTest` looks for the data; I think you forgot to update one of them. I prefer the one where the data is in its own folder. Also, please add the generated data files to `.gitignore` - they appear in my list of untracked files after generation.